spec/features/customers/orders/customer_can_view_previous_orders_details_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+feature "the customer views the details for an order" do
+  let!(:customer) { logged_in_as_customer }
+  let!(:order_1) { create(:all_new_order) }
+  let!(:order_2) { create(:all_new_order) }
+  let!(:order_3) { create(:all_new_order) }
+
```

Comment:
@Robbie-Smith I like the clean up in general - nicer to read the tests and see what they are looking for.  I do think you'll need to rework the setup though.  The `total_price_in_cents` for the order won't save properly the way you're creating the orders.  Basically, the items need to be set when the order is created.  Something like...
`let!(:order) { create(:order, items: [item_1, item_2]) }` 
or however you'd like to declare the items, but basically they need to be declared when the order is created.  Does that make sense?  I really like the way the refactor is going - and I'm happy to chip in to get it cleaned up.